apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.js

```diff
@@ -122,7 +124,9 @@ function WelcomeTour() {
 					[]
 				),
 			],
-			className: 'wpcom-editor-welcome-tour',
+			className: classNames( 'wpcom-editor-welcome-tour', {
+				'is-desktop': ! isMobile(),
```

Comment:
We can add both `is-mobile` & `is-desktop` classes to the tour kit directly, as these are fairly generic concepts that any implementation could benefit from. (could be attached to `tour-kit-frame` - but please check if another parent is better)

p.s. if we can do the above in a PR (this or another) without the welcome-tour modifications, then even better. So one to introduce the viewport classes, and another to follow up on the `absolute` positioning part. In fact, this PR (or whichever "welcome-tour" PR we use) could really instead just remove the `absolute` positioning and focus on the proper fix instead.

Reply:
Thank you @chriskmnds! 😄 
That's an interesting idea, I'll create another PR for introducing the viewport classes, this one instead will be used to fix the actual problem 🚀